subsystem/drivetrain.py

```diff
@@ -13,17 +25,90 @@
 )
 
 class SparkMaxSwerveNode(SwerveNode):
-    m_move: SparkMax
-    m_turn: SparkMax
-    encoder: ctre.CANCoder
-    absolute_encoder_zeroed_pos: float = 0  # Radians
-    drive_reversed: bool = False
-    turn_reversed: bool = False
+
```

Comment:
Again, I think the typed version from before is far more readable than using an init constructor, at least for this scenario.